conan1.0.rst

```diff
@@ -1,15 +1,14 @@
 .. _conan_1_0:
 
 
-Testing conan 1.0.0-beta
-==========================
+Upgrading to conan 1.0
+======================
 
-Conan 1.0 is really close. And we need your help to polish and tune it, before it is released. Once it is released, there will be a strong commitment on not breaking, and changing things will require more time. So we have released a 1.0.0-beta version, you can install and try now:
+If you were using a 0.X conan version, there are some things to consider while upgrading. They are reflected in the changelog, but this tries to summarize the most important changes here:
 
-.. code-block:: bash
 
-    $ pip install conan==1.0.0b3 --upgrade
-    $ conan --version
+Command line changes
+--------------------
 
 There has been a few things that will break existing usage (compared to 0.30). Most of them are in the command line arguments, so they be relatively easy to fix. The most important one is that now most command requires the path to the conanfile folder or file, instead of using ``--path`` and ``--file`` arguments. Specifically, ``conan install``, ``conan export`` and ``conan create`` will be the ones most affected:
```

Comment:
"Most of them are in ~the~ command line arguments, so they **WOULD** be relatively easy to fix"

"now most command**S** require~s~"

Reply:
Maybe stick to new style guidelines: https://github.com/conan-io/docs/tree/develop#style-guidelines?

Should we link the changelog to invite users to see all (breaking) changes?